server/pkg/asset/builder_test.go

```diff
@@ -122,6 +123,21 @@ func TestBuilder_Build(t *testing.T) {
 			},
 			err: ErrZeroSize,
 		},
+		{
+			name: "fail: invalid threadId",
+			input: Input{
+				id:          aid,
+				project:     pid,
+				createdBy:   uid,
+				fileName:    "hoge",
+				size:        size,
+				previewType: PreviewTypeFromRef(lo.ToPtr(PreviewTypeImage.String())),
+				file:        &f,
+				uuid:        "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+				thread:      ThreadID{},
+			},
+			err: ErrZeroSize,
```

Comment:
This test name and `err: ErrZeroSize` is mismatched. Does it test `ErrNoThread`?

Reply:
This is done by mistake, I pushed a fix. Thanks!